helix-core/src/test/java/org/apache/helix/integration/task/TestTaskThrottling.java

```diff
@@ -87,15 +97,17 @@ public void testTaskThrottle() throws Exception {
     _gSetupTool.getClusterManagementTool().setConfig(scope, properties);
 
     String jobName2 = "Job2";
-    Workflow flow2 = WorkflowGenerator.generateSingleJobWorkflowBuilder(jobName2, jobConfig).build();
+    Workflow flow2 = WorkflowGenerator.generateSingleJobWorkflowBuilder(jobName2, jobConfig)
+        .setWorkflowConfig(wfg)
+        .build();
     _driver.start(flow2);
     _driver.pollForJobState(flow2.getName(), TaskUtil.getNamespacedJobName(flow2.getName(), jobName2),
         TaskState.IN_PROGRESS);
 
     // Expect 10 tasks
     Assert.assertTrue(TestHelper.verify(
         () -> (countRunningPartition(flow2, jobName2) == (_numNodes * perNodeTaskLimitation)),
-        TestHelper.WAIT_DURATION));
+        2 * TestHelper.WAIT_DURATION));
```

Comment:
Now even when you change it to 60 seconds, is this 2 * necessary?

Reply:
fixed.